_test/stackalize

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bash
+
+shopt -s extglob
+
+exampleFile="example.hs"
+testSuffix="_test.hs"
+srcDir="src"
+testDir="test"
+programName=`basename "$0"`
+
+depsFinder="`dirname ${BASH_SOURCE}`/dependencies"
+
+generateStackYaml () {
+    echo "resolver: ${stackResolver}"
+}
+
+generatePackageYaml () {
+    local exercise=`basename ${PWD}`
+    local commonDeps=`${depsFinder} --examples-and-tests ${exercise}`
+    local exampleDeps=`${depsFinder} --only-in-examples ${exercise}`
+    local testDeps=`${depsFinder} --only-in-tests ${exercise}`
+
+    echo "name: $exercise"
+    echo ""
+    echo "dependencies:"
+
+    for dependency in ${commonDeps}; do
+      echo "  - ${dependency}"
+    done
+
+    echo ""
+    echo "library:"
+    echo "  source-dirs: ${srcDir}"
+    echo "  dependencies:"
+
+    for dependency in ${exampleDeps}; do
+      echo "    - ${dependency}"
+    done
+
+    echo ""
+    echo "tests:"
+    echo "  test:"
+    echo "    main: ${exercise}${testSuffix}"
+    echo "    source-dirs: ${testDir}"
+    echo "    dependencies:"
+    echo "      - ${exercise}"
+
+    for dependency in ${testDeps}; do
+      echo "      - ${dependency}"
+    done
+}
+
+stackalizeProject () {
+    local exercise=`basename ${PWD}`
+    local module=`grep "^module" ${exampleFile} | cut -d " " -f 2`
+    if [ ! -e "${exampleFile}"           ] ||
+       [ ! -e "${exercise}${testSuffix}" ] ; then
+       echo "$0: Cannot stackalize. Exercise files not found."
+       exit 1
+    else
+        mkdir .extraFiles                                    &&
+        mv * .extraFiles                                     &&
```

Comment:
hmm, would you explain to me what happens if not moving files to here? What would break? Maybe worth a comment here, unless you think it's obvious to anyone who knows what they doing (I certainly don't with Stack, I know)